sources/odoo.conf

```diff
@@ -4,7 +4,7 @@
 ; --------------------
 ; change this ASAP!!!!
 ;---------------------
-admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+admin_passwd = _ADMIN_PASSWORD_
 
 ; set the default timezone for openerp interface
 timezone = Europe/Paris
```

Comment:
As a general comment, I think we should be able to fine-tune the configuration for (almost) every variable. I'm still not sure if the best way is:

1. to use ENV variables in the d-c.yml (+++: very convenient/user friendly for people using our image, ---: need to maintain the "binding" of every variable in the code)
2. to allow ADD odoo.conf in the project specific Dockerfile in order to manage parameters "manually"